### PR TITLE
✨ Implement new NFT details page

### PR DESCRIPTION
**Pull request by nwingt**
image

### CODE REVIEW

File: src/pages/nft/class/_classId/_nftId.vue [hunk 1]
```diff
@@ -104,17 +77,86 @@
...
+            <div
+              v-if="nftCreatorMessage"
+              class="py-[40px] px-[24px] w-full"
+            >
+              <NFTMessage
+                class="w-full"
+                type="purchase"
+                from-type="creator"
+                :from-wallet="iscnOwner"
+                :message="nftCreatorMessage"
```
Comment (williamchong) [inline src/pages/nft/class/_classId/_nftId.vue:140]: in API nftCreatorMessage is a template string that has a `.replace` element (currently not used), should we implement it in client also?

Comment (nwingt) [reply src/pages/nft/class/_classId/_nftId.vue:140]: Implement? Isn't the return memo already parsed?

Comment (williamchong) [reply src/pages/nft/class/_classId/_nftId.vue:137]: but `nftCreatorMessage` is just `return this.NFTClassMetadata?.message || '';` here without parsing?

Comment (nwingt) [reply src/pages/nft/class/_classId/_nftId.vue:137]: I thought the parsing is done during mint, am I getting it wrong?

Comment (nwingt) [reply src/pages/nft/class/_classId/_nftId.vue:137]: Wait, I got it, the parsing is done during collect not mint, so this shows the preview of the message, right?

Comment (williamchong) [reply src/pages/nft/class/_classId/_nftId.vue]: yes, parsing is done during transaction and result is written into memo
so if we really start using template string in message, we might need to implement parsing here too
or we can also ignore this too, the template string is kind of readable

Comment (nwingt) [reply src/pages/nft/class/_classId/_nftId.vue]: We should put the logic into one lib when we develop this further

**Comment by nwingt** [comment]
Merge this into `feature/pnft` branch first until we are ready to launch

**Pull request merged**